src/app/common/duck/operations.ts

```diff
@@ -1,9 +1,31 @@
-import { Creators } from './actions';
-const alertSuccess = Creators.alertSuccess;
-const alertError = Creators.alertError;
-const alertClear = Creators.alertClear;
+import { alertSuccess, alertError, alertClear } from './actions';
+const alertErrorTimeout = message => {
```

Comment:
The arguments to these operations should also only accept TS strings, i.e. `message: String`.